training/Extending and Customising Modules/3-Creating a Subtype Table/3-Common issues and troubleshooting/3-Common issues and troubleshooting.md

```diff
@@ -9,11 +9,14 @@
     - <a href="https://rapiddocs.z8.web.core.windows.net/docs/Rapid/Keyper%20Manual/Designer/Menus/menu-button-configuration/how-to-crud-a-menu-item/" target="_blank">How to **delete** a Menu item</a>
 
 3. Return to **Explorer**.
-4. Open the **Assets** table from the sidebar. Notice that the other asset subtype tables (**IT Assets** and **Vehicle Assets**) are displayed as tabs along the tab strip at the top. Let's add our new table to this tab strip.
+4. Open the **Assets** table from the sidebar.
 
-    - <a href="https://rapiddocs.z8.web.core.windows.net/docs/Rapid/Keyper%20Manual/Designer/Pages/Layouts/list-of-available-layouts/#tab-strip" target="_blank">How to add a new tab to the Tab Strip</a>
+### Editing the Tab Strip
+Notice that the other asset subtype tables (**IT Assets** and **Vehicle Assets**) are displayed as tabs along the tab strip at the top. Let's add our new table to this tab strip.
 
-### Building the Create and Details Layout for a Table
+- <a href="https://rapiddocs.z8.web.core.windows.net/docs/Rapid/Keyper%20Manual/Designer/Pages/Layouts/list-of-available-layouts/#tab-strip" target="_blank">How to add a new tab to the Tab Strip</a>
```

Comment:
The href can be replaced with `http://docs.rapidplatform.com/docs...` now as we've shifted domains
I think I've fixed this in a few places in training so far? Not sure exactly where things are until we merge-inate things

Reply:
Done!